### ISSUE TITLE
Create a script that convert from json to other formats

### ISSUE THREAD

**Issue by seddik** [bensialih/algeria-cities#180]
we need a script to convert from json to other formats automatically so when we add a new feature or field to json, other formats stays up to date.

**Comment by bensialih** [comment]
Hi @seddik 
Can you check the csv output for converting from json to csv on this draft PR
https://github.com/bensialih/algeria-cities/pull/185

So my process is to create a script that migrates from the original json to the original csv.
This is to make sure that we have data fidelity and consistency. We dont want to introduce any breaking changes.
After this we can run a script with the osm_ids that will output to csv and sql.

The thing that Im noticing is that some entries are quoted and have a space at the beginning of things like wilaya_name
`41,الحجاج,El Hadjadj,أولاد بن عبد القادر,Ouled Ben Abdelkader,02," الشلف",Chlef`

Please can you check this and confirm that you are ok overwriting this and having the name that has removed any trailing spaces.

**Comment by seddik** [comment]
@bensialih  yes I suppose we need to remove the quotes and trim the spaces on both sides (left and right), you can add replacing double spaces with one space if there is that case. so we would have clean data.
